src/main/kotlin/three/consulting/epoc/service/FirebaseService.kt

```diff
@@ -0,0 +1,63 @@
+package three.consulting.epoc.service
+
+import com.google.auth.oauth2.GoogleCredentials
+import com.google.firebase.FirebaseApp
+import com.google.firebase.FirebaseOptions
+import com.google.firebase.auth.FirebaseAuth
+import mu.KotlinLogging
+import org.springframework.stereotype.Service
+import three.consulting.epoc.common.Role
+import three.consulting.epoc.dto.EmployeeDTO
+import three.consulting.epoc.dto.FirebaseUserDTO
+import three.consulting.epoc.entity.Employee
+import three.consulting.epoc.repository.EmployeeRepository
+
+private val logger = KotlinLogging.logger {}
+
+@Service
+class FirebaseService(
+    private val employeeRepository: EmployeeRepository
```

Comment:
If you configure the firebaseAuth() method as a bean, you can just add in the constructor here after the repository
```kotlin
private val firebaseAuth: FirebaseAuth
```